draft-ietf-quic-ack-frequency.md

```diff
@@ -535,11 +535,10 @@ Note that the congestion window and the RTT estimate change over the lifetime of
 connection and therefore might require sending updates in an ACK_FREQUENCY frames to
 ensure optimal performance, though not every change should trigger an update.
 Usually, it is not necessary to send an ACK_FREQUENCY frame more than once per
-RTT and likely it needs to be sent even less frequently.
-Ideally, an ACK_FREQUENCY frame is sent only when a relevant change
-in the congestion window or smoothed RTT is detected that impacts the local
-setting of the reordering threshold or locally-selected calculation of the
-either Ack-Eliciting Threshold or the Requested Max Ack Delay.
+RTT and probably even less frequently. Ideally, an ACK_FREQUENCY frame is sent only
```

Comment:
Sorry for being super nit-picking here but we spend soo much time to word-smith this part that I want to be super careful. Is there a reason why you changed "likely" to "probably"? As a non-native speaker there is not much difference for me but maybe "likely" is slightly stronger? Not a big issues but wanted to double-check...

Reply:
Good point, likely is better here.